app/login/page.tsx

```diff
@@ -0,0 +1,104 @@
+"use client";
+
+import { useState } from "react";
+import { useRouter, useSearchParams } from "next/navigation";
+import { Input } from "../[eventSlug]/input";
+import clsx from "clsx";
+
+interface LoginResponse {
+  success: boolean;
+  error?: string;
+  redirect?: string;
+}
+
+export default function LoginPage() {
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+  const router = useRouter();
+  const searchParams = useSearchParams();
+  const redirectTo = searchParams?.get("redirect") || "/";
+
+  const handleSubmit = async (e: React.FormEvent) => {
```

Comment:
The form submit should be a server action. See the other comments for the implementation.